dll/win32/shell32/CShellLink.cpp

```diff
@@ -2802,6 +2802,29 @@ BOOL CShellLink::OnInitDialog(HWND hwndDlg, HWND hwndFocus, LPARAM lParam)
     if (m_sDescription)
         SetDlgItemTextW(hwndDlg, IDC_SHORTCUT_COMMENT_EDIT, m_sDescription);
 
+    /* Hot key */
+    SendDlgItemMessageW(hwndDlg, IDC_SHORTCUT_KEY_HOTKEY, HKM_SETHOTKEY, m_Header.wHotKey, 0);
+
+    /* Run */
+    const WORD runstrings[] = { IDS_SHORTCUT_RUN_NORMAL, IDS_SHORTCUT_RUN_MIN, IDS_SHORTCUT_RUN_MAX };
+    const DWORD runshowcmd[] = { SW_SHOWNORMAL, SW_SHOWMINNOACTIVE, SW_SHOWMAXIMIZED };
+    HWND hRunCombo = GetDlgItem(hwndDlg, IDC_SHORTCUT_RUN_COMBO);
+    for (UINT i = 0; i < _countof(runstrings); ++i)
+    {
+        WCHAR buf[MAX_PATH];
+        if (!LoadStringW(shell32_hInstance, runstrings[i], buf, _countof(buf)))
+            break;
+
+        int index = SendMessageW(hRunCombo, CB_ADDSTRING, 0, (LPARAM)buf);
+        if (index < 0)
+            continue;
+        SendMessageW(hRunCombo, CB_SETITEMDATA, index, runshowcmd[i]);
+        if (!i || m_Header.nShowCommand == runshowcmd[i])
+        {
+            SendMessageW(hRunCombo, CB_SETCURSEL, index, 0);
+        }
```

Comment:
Style consistency with two other if's above:
```suggestion
            SendMessageW(hRunCombo, CB_SETCURSEL, index, 0);
```